assimilation_code/modules/assimilation/filter_mod.f90

```diff
@@ -2345,13 +2357,12 @@ function count_state_ens_copies(ens_size, post_inflate, prior_inflate) result(nu
 CURRENT_COPIES    = (/ ENS_MEM_START, ENS_MEM_END, ENS_MEAN_COPY, ENS_SD_COPY, &
                        PRIOR_INF_COPY, PRIOR_INF_SD_COPY, POST_INF_COPY, POST_INF_SD_COPY /)
 
-! CSS If Whitaker/Hamill (2012) relaxation-to-prior-spread (rpts) inflation (inf_flavor = 4)
-!  then we need an extra copy to hold (save) the prior ensemble spread
-!   ENS_SD_COPY will be overwritten with the posterior spread before
-!   applying the inflation algorithm; hence we must save the prior ensemble spread in a different copy
-if ( inf_flavor(2) == 4 ) then ! CSS
-   SPARE_PRIOR_SPREAD = next_copy_number(cnum)
-endif 
+! If Whitaker/Hamill (2012) relaxation-to-prior-spread (rpts) inflation (inflation_flavor = 4)
+! then we need an extra copy to hold (save) the prior ensemble spread
+! ENS_SD_COPY will be overwritten with the posterior spread before
+! applying the inflation algorithm; hence we must save the prior ensemble spread in a different copy
+
+if (do_rtps_inflate(post_inflate)) SPARE_PRIOR_SPREAD = next_copy_number(cnum)
```

Comment:
### WHY IS THIS NOT WORKING?!

the `do_rtps_inflate(post_inflate)  call is returning FALSE even though `inf_flavor(2) == 4`

Reply:
Don't care anymore ... reverted to previous method.

Reply:
don't forget to change filter_mod.dopplerfold.f90 accordingly if you're updating filter_mod.f90